x-pack/elastic-agent/pkg/basecmd/version/cmd.go

```diff
@@ -64,7 +64,7 @@ func NewCommandWithArgs(streams *cli.IOStreams) *cobra.Command {
 			binaryOnly, _ := cmd.Flags().GetBool("binary-only")
 			if !binaryOnly {
 				if d, err := queryDaemon(); err != nil {
-					returnErr = fmt.Errorf("failed to communicate with running daemon: %w", err)
+					returnErr = fmt.Errorf("could not get version. failed to communicate with running daemon: %w\n Use --binary-only flag to get version from binary.", err)
```

Comment:
I would update that to so

`Use --binary-only flag to skip trying to retrieve version from running daemon.`

Reply:
updated